src/EnergyPlus/DataEnvironment.cc

```diff
@@ -159,50 +159,6 @@ Real64 OutWetBulbTempAt(EnergyPlusData &state, Real64 const Z) // Height above g
     return LocalOutWetBulbTemp;
 }
 
-Real64 OutDewPointTempAt(EnergyPlusData &state, Real64 const Z) // Height above ground (m)
-{
-
-    // FUNCTION INFORMATION:
-    //       AUTHOR         Linda Lawrie
-    //       DATE WRITTEN   March 2007
-    //       MODIFIED       na
-    //       RE-ENGINEERED  na
-
-    // PURPOSE OF THIS FUNCTION:
-    // Calculates outdoor dew point temperature at a given altitude.
-
-    // METHODOLOGY EMPLOYED:
-    // 1976 U.S. Standard Atmosphere.
-    // copied from outwetbulbtempat
-
-    // REFERENCES:
-    // 1976 U.S. Standard Atmosphere. 1976. U.S. Government Printing Office, Washington, D.C.
-
-    // Return value
-    Real64 LocalOutDewPointTemp; // Return result for function (C)
-
-    // FUNCTION LOCAL VARIABLE DECLARATIONS:
-    Real64 BaseTemp; // Base temperature at Z = 0 (C)
-
-    BaseTemp = state.dataEnvrn->OutDewPointTemp + state.dataEnvrn->WeatherFileTempModCoeff;
-
-    if (state.dataEnvrn->SiteTempGradient == 0.0) {
-        LocalOutDewPointTemp = state.dataEnvrn->OutDewPointTemp;
-    } else if (Z <= 0.0) {
-        LocalOutDewPointTemp = BaseTemp;
-    } else {
-        LocalOutDewPointTemp = BaseTemp - state.dataEnvrn->SiteTempGradient * DataEnvironment::EarthRadius * Z / (DataEnvironment::EarthRadius + Z);
-    }
-
-    if (LocalOutDewPointTemp < -100.0) {
-        ShowSevereError(state, "OutDewPointTempAt: outdoor dewpoint temperature < -100 C");
-        ShowContinueError(state, format("...check heights, this height=[{:.0R}].", Z));
-        ShowFatalError(state, "Program terminates due to preceding condition(s).");
-    }
-
-    return LocalOutDewPointTemp;
-}
-
```

Comment:
[src/EnergyPlus/DataEnvironment.cc:162]:(style),[unusedFunction],The function 'OutDewPointTempAt' is never used.

Reply:
:+1: :+1: :+1: